backend/corpora/dataset_processing/process.py

```diff
@@ -122,19 +124,18 @@ def update_db(metadata=None, processing_status=None):
         dataset.update(processing_status=status)
 
 
-def fetch_dropbox_url(dropbox_url, local_path):
+def fetch_dropbox_url(dataset_uuid: str, dropbox_url: str, local_path: str) -> str:
```

Comment:
can you update the name/signature of this function to make it clear that it kicks off the upload process? And maybe add a test to make it clear that the function always returns the local path you pass in as an arg? Also why does it need to return the local path at all?

Reply:
I update the name